salt/modules/junos.py

```diff
@@ -654,8 +652,10 @@ def rollback(**kwargs):
         ids_passed = ids_passed + 1
 
     if ids_passed > 1:
-        log.warning("junos.rollback called with more than one possible ID.")
-        log.warning("Use only one of the positional argument, `id`, or `d_id` kwargs")
+        log.warning(
+            "junos.rollback called with more than one possible ID. "
+            "Use only one of the positional argument, `id`, or `d_id` kwargs"
+        )
```

Comment:
**praise**: This is a good change to move to a single warning. At first I was :suspect: but reading through it really is just the one statement, so :+1: